### DISCUSSION TITLE
trend.json 안에 데이터 값 질문

### DISCUSSION THREAD

**Discussion by baechanghyeon** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#27] [General]
trend.json 안에 데이터 값 정의 해봤습니다. 혹시 틀린 부분이나 애매한 부분은 말씀해주세요 ! 

- imp: 광고가 보여진 횟수
- click: 클릭한 횟수
- cost: 금액 ( 매출 인지 광고비인지... cost 자체가 비용이라서 광고비 인거 같기도 합니다 )
- convValue: 전환된 값 매출액 ( ? )
- conv: 전환 수 ( ? ) 
- ctr: 클릭률
- cvr: 전환률
- cpc: 클릭 당 비용
- cpa: 광고주가 원하는 행동시 비용을 지불
- roas: 광고비에 대한 매출 비율 
- date: 날짜 

일단 데이터 값들을 이렇게 정의했는데 애매한 부분은 ( ? ) 달았습니다. 
혹시 다른 의견있으시면 코멘트 부탁드립니다.

**Comment by dldnjswns31** [comment]
필요한 데이터 ROAS / cost(광고비) / imp( 노출수) / click(클릭수) / conv(전환수) / convValue(매출) 이라고 생각됩니다.

**Reply by baechanghyeon** [reply]
roas = convValue / cost * 100 계산해보니까 convValue => 매출이 맞네요 ! 감사합니다
